playbooks/common/openshift-cluster/upgrades/init.yml

```diff
@@ -9,6 +9,15 @@
 - import_playbook: ../../../init/base_packages.yml
 - import_playbook: ../../../init/cluster_facts.yml
 
+- name: Inspect cluster certificates
+  hosts: "{{ l_upgrade_cert_check_hosts }}"
+  tasks:
+  - import_role:
+      name: openshift_certificate_expiry
+      tasks_from: main.yml
+    vars:
+      openshift_certificate_expiry_warning_days: 10000
```

Comment:
Needs to be configurable?

Reply:
Yeah, also make sure to mention it in inventory/hosts.example and default it to 365.